extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -211,7 +211,7 @@ void ListenHTTP::onSchedule(core::ProcessContext *context, core::ProcessSessionF
 
 ListenHTTP::~ListenHTTP() = default;
 
-void ListenHTTP::onTrigger(core::ProcessContext *context, core::ProcessSession *session) {
+void ListenHTTP::onTrigger(core::ProcessContext* /*context*/, core::ProcessSession *session) {
```

Comment:
I would remove commented variables where the parameter is self-explanatory or has even less meaning than the typename, like ProcessContext - context, CivetServer - server, Configure - configure, ProcessSessionFactory - sessionFactory, ProcessGroup - pg, Parameters params, Repository - repo and so on. If you have an idea it could be filtered and replaced easily go ahead, if it would be too exhausting, just leave it for now.

Reply:
I don't have a quick-fix for that. People probably need to edit those cases one-by-one when they encounter them.